arctic/store/bson_store.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, arctic_lib):
     def initialize_library(cls, arctic_lib, hashed=True, **kwargs):
         logger.info("Trying to enable sharding...")
         try:
-            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed)
+            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed, key='_id')
```

Comment:
More a mention FYI.  We set hashed=True on everything as there's little downside.